news/2017-07-07-scorewatch-july-week-1.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Scorewatch: July Week 1"
+permalink: http://osunews.tumblr.com/post/162432090863/scorewatch-july-week-1
+date: 2017-07-07 15:00:00 +0200
+tumblr_url: http://osunews.tumblr.com/post/162432090863/scorewatch-july-week-1
+---
+A fierce war burns brightly in the top 5 of osu!standard, a Glitch Nerds score battle between the 2 best players of osu!mania, and some taiko action all wrapped up in the first week of July!
```

Comment:
A fierce war in the top 5 of osu!standard, a Glitch Nerds score battle between the 2 best players of osu!mania, and some taiko action all wrapped up in the first week of July!